src/_test_/components.test.js

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Home from '../spa/home';
+import Quote from '../Api/Quote';
+import Calculator from '../components/Calculator';
+
+describe('Tests for Navigation components', () => {
+  test('Case 1: Home page snapshot', () => {
+    const check = render(<Home />);
+    expect(check).toMatchSnapshot();
+  });
```

Comment:
> - [ ] Good job so far @the-faizmohammad, I noticed that one of your test cases for components is failing. Please could you address this issue? Having all test cases pass is a good practice because it helps to improve the quality, reliability, and maintainability of your code.👍
> 
> ![ts](https://github.com/the-faizmohammad/MathsMagicians/assets/40009354/0e61490e-d17e-4939-8563-b4edc2e432fc)
>